enterprise/signals.py

```diff
@@ -135,6 +136,57 @@ def delete_enterprise_learner_role_assignment(sender, instance, **kwargs):     #
             pass
 
 
+@receiver(post_save, sender=EnterpriseCustomerCatalog)
+def update_enterprise_catalog_data(sender, instance, **kwargs):     # pylint: disable=unused-argument
+    """
+    Send data changes to Enterprise Catalogs to the Enterprise Catalog Service.
+    """
+    catalog_uuid = instance.uuid
+    try:
+        catalog_client = EnterpriseCatalogApiClient()
+        response = catalog_client.get_enterprise_catalog(catalog_uuid)
+    except NotConnectedToOpenEdX as exc:
+        logger.exception('Unable to update Enterprise Catalog {}'.format(str(catalog_uuid)), exc)
+    else:
+        if kwargs['created'] or not response:
```

Comment:
I know it shouldn't happen, but what if `kwargs['created']` is true _and_ we get a response?